readme.md

```diff
@@ -85,6 +85,12 @@ linkifyUrls('See https://sindresorhus.com/foo', {
 //=> 'See <a href="https://sindresorhus.com/foo">/foo</a>'
 ```
 
+## Compatibility note
+
+Since v2.2.0, the main regular expression is using a negative look-behind to address [#7](https://github.com/sindresorhus/linkify-urls/issues/7).
+Look-behind assertions [should be compatible since Node v6](https://node.green/#ES2018-features--RegExp-Lookbehind-Assertions) (with the use of `--harmony` flag).
+At the time of this note, only Google Chrome seems to support it, even though it [will eventually be a standard](https://tc39.github.io/ecma262/).
```

Comment:
Thanks for writing this. I'm gonna slim it down a little bit though as it's quite verbose.